I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 165, in script
    python:
  File "game/script.rpy", line 166, in <module>
    renpy.show(rain)
AttributeError: 'Particles' object has no attribute 'split'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 165, in script
    python:
  File "/Users/kids/Desktop/renpy-8.3.7-sdk/renpy/ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/kids/Desktop/renpy-8.3.7-sdk/renpy/python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script.rpy", line 166, in <module>
    renpy.show(rain)
  File "/Users/kids/Desktop/renpy-8.3.7-sdk/renpy/exports/displayexports.py", line 442, in show
    name = tuple(name.split())
AttributeError: 'Particles' object has no attribute 'split'

macOS-10.16-x86_64-i386-64bit x86_64
Ren'Py 8.3.7.25031702
no right answer 1.0
Sat Sep 27 15:30:57 2025
